.github/workflows/android-cd.yml

```
@@ -0,0 +1,68 @@
+          path: android/app/build/outputs/bundle/release/app-release.aab
+          overwrite: 'true'
+
+      - name: Deploy AAB On Google Play Console
+        uses: r0adkll/upload-google-play@v1.1.3
+        with:
+          serviceAccountJsonPlainText: ${{ secrets.GOOGLE_SERVICE_ACCOUNT_KEY }}
+          packageName: com.onair.hearit
+          releaseFiles: android/app/build/outputs/bundle/release/app-release.aab
+          track: alpha
+          whatsNewDirectory: android/app/release_note.txt
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)